tools/fds/server.cpp

```diff
@@ -269,10 +270,8 @@ int main (
         cout << endl << "### Server shut down ###" << endl;
```

Comment:
I would delete the participant here:

```suggestion
        cout << endl << "### Server shut down ###" << endl;
        DomainParticipantFactory::get_instance()->delete_participant(pServer);
```

Reply:
added